### ISSUE TITLE
Console error for missing fontawesome stylesheet

### ISSUE THREAD

**Issue by JasonEtco** [github/training-kit#545]
Started in https://github.com/github/services-site/pull/43, there is a console error that appears to be looking for a fontawesome stylesheet that doesn't actually exist in the repo.

```
[2017-08-08 15:56:42] ERROR/on-demand/assets/fonts/font-awesome/css/font-awesome.min.css' not found.
```

This `<link>` tag should be removed from the `index.html` file.

**Comment by wisn** [comment]
@JasonEtco The link you're pointed out is broken.

**Comment by cg-cnu** [comment]
Seems like something I can handle but the given link is broken. 
Would you mind fixing it and give more details :)

**Comment by cg-cnu** [comment]
@wisn you just beat me to it!

**Comment by brianamarie** [comment]
@JasonEtco @wisn @cg-cnu Ah, since the link in the original post is to a private repository, it isn't publicly viewable. Some info about that link: 

Body of the original PR: 

```
This PR removes this error when building locally: [2017-08-08 15:56:42] ERROR/on-demand/assets/fonts/font-awesome/css/font-awesome.min.css' not found.`

This file wasn't being found aka it wasn't being used aka the file actually doesn't exist, so I removed the code from the index.html file so that error wouldn't happen anymore.
```

Content of the PR: Just removed that link from the index, a 1 line removal. Very simple. 😄 

Hopefully this context is helpful. If anyone would like to take it on it would be very appreciated, but please feel free to ask if you have any more questions. 👍

**Comment by wisn** [comment]
@cg-cnu If you want to handle this and fix it, I don't mind.

**Comment by cg-cnu** [comment]
@wisn That's completely cool! Please go ahead. Good luck! 😄

**Comment by JasonEtco** [comment]
Woops sorry about that friends 🙇 Thanks for clearing it up @brianamarie

**Comment by JasonEtco** [comment]
Actually, I'm going to close this because it's been addressed in a9414ebe033f628205b9c1f59f6efd71316c0403. Thanks everyone ❤️
